Parity/prminput/prex_maindet_pedestal.3131-3423.map

```diff
@@ -0,0 +1,21 @@
+!Man detector and AT calibrations from run 3138, using cameron's analysis http://ace.phys.virginia.edu/HAPPEX/4095
+ usl, 1383.97, 0.000076
+ usr, 1300.45, 0.000076
+ dsl, 4306.02, 0.000886
```

Comment:
Sorry about this... The gains for main detectors should always be 0.000076. I had left the "real" gain output in the "all-plots" outputs, and this run for some reason failed to show up in the "out/" folder.

Anyway, the usl and usr have the right normalization, but we may want to fix the DS ones too